src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import * as FaIcons from "react-icons/fa";
+
+export const MenuItems = [
```

Comment:
I have a few suggestions here. First of all, the file should be called `MenuItem` and have `.jsx` format. In shortcut, it should be a component that has the code for a single menu list item component.

Second, the `MenuItems` constant should be called `MENU_ITEMS` and should be moved to the `Navbar` component.